help/en/html/hardware/loconet/SE74.shtml

```diff
@@ -0,0 +1,577 @@
+                flashing aspects and "SE8C" mode signaling.
+            <li>If you want/need control over the AUX outputs on the S1 thru S4 
+                cables, you (apparently) must use Aspect mode, as there appears 
+                to be no way to control the AUX outputs in "SE8C Compatible" mode.  
+                JMRI does not have any reasonable way to control the AUX output 
+                (i.e. what SE74 documentation calls the "Lunar" aspect) when 
+                SE74 OpSw41 is "C"losed. 
+            </li>
+            <li>Because SE8C-mode signaling is controlled via "Turnout" operations,
+                it is necessary to program the SE8C so that its signal control
+                addresses do not "conflict" with any other turnout-controlled 
+                device on the layout.  SE74 "Aspect" mode signaling can only 
+                conflict with those devices which use the "Extended Accessory 
+                Decoder" messaging (or its LocoNet equivalent), thus making it <em>
+                    far</em> easier to void addressing conflicts.</li>
+            <li>Control of a "Lunar" aspect is not available when the SE74 operates 
+            in "SE8C" signaling mode.  When the SE74 operates in "Aspect" signaling 
+            mode, a "Lunar" aspect may be displayed.  Depending on the SE74 OpSw 
+            settings, the "Lunar" aspect may be displayed in addition to another 
+            aspect (Red/Yellow/Green/Flashing Red/Flashing Yellow/Flashing Green), 
+            or instead of that other aspect.</li>
+        </ul>
+        <p>In the author's <em>opinion</em>, "Aspect" mode is preferred over "SE8C 
+            Compatible mode" for any system which can support "Aspect" mode.
+            
+        <h3 id="configAspectSignaljmri">Configuring SE74 "Aspect"-mode "Signal Head" objects in JMRI</h3>
+
+        <p>This section describes how to associate a JMRI "Signal Head" object with the
+        SE74 hardware and one of its controlled Signal Heads, when the SE74 is 
+        configured to use "Aspect"-mode signaling.  This section does not apply 
+        when the SE74 is configured for "SE8C"-mode signaling; see <a href="#configSe8cSignalingjmri">
+        this section</a> for information on configuring JMRI Signal Head objects 
+        when using the SE74 in "SE8C"-mode.
+        
+        <p>It is assumed that you have JMRI configured to communicate with your 
+        DCC system, and that you have configured the SE74 for "Aspect" mode signaling
+        and that the SE74 has a base address which does not conflict with any other 
+        aspect-mode signaling hardware.  With those assumptions met, you may 
+        start configuring the JMRI "Signal Head" objects.
+        
+        <p>Within the JMRI Panel Pro application, open the Signal Heads" table 
+        (i.e. "Tools"-"Tables"-"Signal Heads").  This table shows all of the 
```

Comment:
Note the comment from @dsand47 that it's recommended to use `&rArr;`, so this should be:

`<strong>Tools &rArr; Tables &rArr; Signal Heads</strong>`

or

`<strong>Tools&nbsp;&rArr; Tables&nbsp;&rArr; Signal Heads</strong>`

The latter uses `&nbsp;` to prevent line break before the arrow.